rotkehlchen/tests/unit/test_stats.py

```diff
@@ -103,3 +104,18 @@ def test_compound_events_stats(
     # Check rewards mappings
     rewards_0 = stats['rewards']['0x65304d6aff5096472519ca86a6a1fea31cb47Ced']
     assert rewards_0[A_COMP].amount == FVal('0.06366460157923238')
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [TEST_ACCOUNTS])
+def test_liquity_tcr(ethereum_inquirer: 'EthereumInquirer'):
```

Comment:
nitpick, it's not the TCR bu the having a non-exact price

```suggestion
def test_liquity_tcr_non_exact_int(ethereum_inquirer: 'EthereumInquirer'):
```